packages/tpetra/core/src/Tpetra_Details_Behavior.cpp

```diff
@@ -22,6 +37,81 @@ namespace { // (anonymous)
     return s;
   }
 
+  void
+  split(const std::string& s,
+        std::function<void(const std::string&)> f,
```

Comment:
I forgot to mention -- you have to include `<functional>` in order to use `std::function`.

Reply:
hmm, built and passed tests on my machine.  but, i'll add it